bin/varnishd/cache/cache_esi.h

```diff
@@ -39,7 +39,8 @@
 #define	VEC_S1	(0x60 + 1)
 #define	VEC_S2	(0x60 + 2)
 #define	VEC_S8	(0x60 + 8)
-#define	VEC_INCL	'I'
```

Comment:
swap `i` and `I`?

Reply:
I'm tempted to use `A` and `C` and retire `I` altogether.

Reply:
'A' is nogo, it is already used for VEC_V1 (0x40+1)

I would prefer we stuck to hex definitions throughout to avoid such issue going forward.